math/src/main/scala/breeze/linalg/support/CanTraverseValues.scala

```diff
@@ -25,9 +25,12 @@ import breeze.math.Complex
  * @author dlwh
  */
 trait CanTraverseValues[From, A] {
-  /**Traverses all values from the given collection. */
+
+  /** Traverses all values from the given collection. */
   def traverse(from: From, fn: ValuesVisitor[A]): Unit
-  def isTraversableAgain(from: From):Boolean
+
+  @deprecated
+  final def isTraversableAgain(from: From): Boolean = true
 
```

Comment:
i thought there was an Iterator implementation of CanTraverseValues? There should be.